wled00/FX.cpp

```diff
@@ -5048,13 +5052,13 @@ uint16_t mode_2DDrift() {              // By: Stepko   https://editor.soulmateli
     int mySin = sin_t(angle) * i;
     int myCos = cos_t(angle) * i;
     SEGMENT.setPixelColorXY(colsCenter + mySin, rowsCenter + myCos, ColorFromPalette(SEGPALETTE, (i * 20) + t_20, 255, LINEARBLEND));
-    if (SEGMENT.check1) SEGMENT.setPixelColorXY(colsCenter + myCos, rowsCenter + mySin, ColorFromPalette(SEGPALETTE, (i * 20) + t_20, 255, LINEARBLEND));
+    if (SEGMENT.check2) SEGMENT.setPixelColorXY(colsCenter + myCos, rowsCenter + mySin, ColorFromPalette(SEGPALETTE, (i * 20) + t_20, 255, LINEARBLEND));
```

Comment:
This will cause *different looking* effect on existing presets. Please consider backwards compatibility.